client/src/components/History/Content/ContentOptions.vue

```diff
@@ -12,7 +12,6 @@ const props = defineProps({
     isVisible: { type: Boolean, default: true },
     state: { type: String, default: "" },
     itemUrls: { type: Object, required: true },
-    keyboardSelectable: { type: Boolean, default: true },
```

Comment:
Why was this removed?

Reply:
The ContentOptions buttons are always Keyboard selectable now, with the Tab key. To go from one item to another, you can use the arrow keys, the tab key will take you from ContentItem -> ContentOptions -> ContentItem ....